python/cudf/cudf/tests/test_rolling.py

```diff
@@ -78,16 +93,16 @@ def test_rolling_dataframe_basic(data, agg, nulls, center):
     pdf = pd.DataFrame(data)
 
     if len(pdf) > 0:
-        for col_name in pdf.columns:
+        for col_idx in range(len(pdf.columns)):
             if nulls == "one":
                 p = rng.integers(0, len(data))
-                pdf[col_name][p] = np.nan
+                pdf.iloc[p, col_idx] = np.nan
             elif nulls == "some":
                 p1, p2 = rng.integers(0, len(data), (2,))
-                pdf[col_name][p1] = np.nan
-                pdf[col_name][p2] = np.nan
+                pdf.iloc[p1, col_idx] = np.nan
```

Comment:
was this just a cosmetic change or something on pandas is throwing a warning for this step?

Reply:
The previous code raised `SettingWithCopyWarning` because it relied on chained indexing. I've been trying to mark the specific warning I'm fixing in each commit message. 36a4ab574fd26171465b5f9b2effbae50fe20a45

There are other ways to fix this warning. For instance, we could use `.loc` if we wanted to use the column name and the dataframe index values instead of plain row/column indices with `.iloc`.

Reply:
Got it, thanks 👍